src/ML-POD/fitpod_command.h

```diff
@@ -0,0 +1,155 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/ Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+
+#ifdef COMMAND_CLASS
+// clang-format off
+CommandStyle(fitpod,CFITPOD);
+// clang-format on
+#else
+
+#ifndef LMP_FITPOD_COMMAND_H
+#define LMP_FITPOD_COMMAND_H
+
+#include "command.h"
+
+namespace LAMMPS_NS {
+
+class CFITPOD : public Command {
+private:
+
+public:
+  struct datastruct {
+    std::string file_format;
+    std::string file_extension;
+    std::string data_path;
+    std::vector<std::string> data_files;
+    std::vector<std::string> filenames;
+
+    std::vector<int> num_atom;
+    std::vector<int> num_atom_cumsum;
+    std::vector<int> num_atom_each_file;
+    std::vector<int> num_config;
+    std::vector<int> num_config_cumsum;
+    int num_atom_sum;
+    int num_atom_min;
+    int num_atom_max;
+    int num_config_sum;
+
+    double *lattice;
+    double *energy;
+    double *stress;
+    double *position;
+    double *force;
+    int *atomtype;
+
+    int training = 1;
+    int normalizeenergy = 1;
+    int training_analysis = 1;
+    int test_analysis = 1;
+    int training_calculation = 0;
+    int test_calculation = 0;
+    int randomize = 1;
+    double percentage = 1.0;
```

Comment:
This seems to be a *fraction* (range 0.0-1.0) and not a *percentage* (range 0.0 to 100.0).

Reply:
_Percentage_ is changed to _fraction_.